dev/MRTCore/azure-pipelines.yml

```diff
@@ -168,7 +168,7 @@ jobs:
     inputs:
       SourceFolder: '$(MRTBinariesDirectory)\Release\AnyCPU\Microsoft.ApplicationModel.Resources.Projection'
       Contents: |
-        Microsoft.ApplicationModel.Resources.winmd
+        net5.0\Microsoft.ApplicationModel.Resources.winmd
```

Comment:
>net5.0 [](start = 8, length = 6)

need to see if this builds

Reply:
nope

---
In reply to: [491236518](https://github.com/microsoft/ProjectReunion/pull/199#discussion_r491236518) [](ancestors = 491236518)

Reply:
On my machine, I see this:

S:\feature_work\reunion_mrt_tooling\MRTCore\mrt\AnyCPU\Debug\Microsoft.ApplicationModel.Resources.Projection\net5.0\Microsoft.ApplicationModel.Resources.winmd

Perhaps the source dir needs to be changed here to
`SourceFolder: '$(MRTBinariesDirectory)\AnyCPU\ReleaseMicrosoft.ApplicationModel.Resources.Projection'`

Also, I wonder why this has worked fine in the CI builds in MSCodeHub...

Reply:
Fixed.